ci_framework/roles/rhol_crc/tasks/main.yml

```diff
@@ -63,33 +63,16 @@
     - name: Configure and start RHOL/CRC
       when: (cifmw_rhol_crc_force_cleanup|bool) or not crc_running|bool
       block:
-        - name: Configure RHOL/CRC using install_yamls
-          when: cifmw_rhol_crc_use_installyamls|bool
-          ansible.builtin.import_tasks: install_yamls.yml
+        - name: Set RHOL/CRC configuration options
+          ansible.builtin.import_tasks: configuration.yml
 
-        - name: Manually configure RHOL/CRC
-          when: not cifmw_rhol_crc_use_installyamls|bool
-          block:
-            - name: Set RHOL/CRC configuration options
-              ansible.builtin.import_tasks: configuration.yml
+        - name: Setup RHOL/CRC
+          ansible.builtin.command: "{{ cifmw_rhol_crc_binary }} setup"
+          register: cifmw_rhol_crc_cmd_setup
 
-            - name: Setup RHOL/CRC
-              ansible.builtin.command: "{{ cifmw_rhol_crc_binary }} setup"
-              register: cifmw_rhol_crc_cmd_setup
-
-            - name: Start RHOL/CRC
-              ansible.builtin.command: "{{ cifmw_rhol_crc_binary }} start"
-              register: cifmw_rhol_crc_cmd_start
-
-    - name: Attach default network to CRC
```

Comment:
This has broken baremetal job.


+ cat
+ IP_ADDRESS_SUFFIX=6
+ CTLPLANE_IP_ADDRESS_SUFFIX=11
error: timed out waiting for the condition on nodenetworkconfigurationpolicies/enp6s0-crc-74q6p-master-0
gmake: *** [Makefile:1888: nncp] Error 1

Reply:
Good catch. I see the same breakage https://github.com/openstack-k8s-operators/tcib/pull/102

Reply:
Yes, see the same error here - https://github.com/openstack-k8s-operators/openstack-baremetal-operator/pull/106#issuecomment-1819814634